ui/src/app/views/notebook-frame/component.ts

```diff
@@ -0,0 +1,44 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {DomSanitizer} from '@angular/platform-browser';
+import {ActivatedRoute} from '@angular/router';
+
+import {WorkspaceData} from 'app/resolvers/workspace';
+import {Workspace} from 'generated';
+
+@Component({
+  selector: 'app-notebook-frame',
+  styleUrls: ['./component.css'],
+  templateUrl: './component.html'
+})
+
+export class NotebookFrameComponent implements OnInit {
```

Comment:
nit: naming ... would this be better named `NotebookIFrameComponent`? I was halfway through the PR before I understood what a frame was.

Reply:
I can rename